BT-Tracking/AppDelegate.swift

```diff
@@ -242,28 +247,42 @@ private extension AppDelegate {
             scannerStore.deleteOldRecordsIfNeeded()
         }
     }
+
+    private func checkFetchedMinSupportedVersion() {
+        if RemoteValues.minSupportedVersion > Version.currentAppVersion {
+            advertiser.stop()
+            scanner.stop()
+            let viewController = UIStoryboard(name: "ForceUpdate", bundle: nil).instantiateViewController(withIdentifier: "ForceUpdateVC")
+            viewController.modalPresentationStyle = .fullScreen
+            window?.rootViewController?.present(viewController, animated: true)
+        }
+    }
     
     private func setupInterface() {
         let window = UIWindow()
         window.backgroundColor = .black
         window.makeKeyAndVisible()
         self.window = window
 
-        let storyboard: UIStoryboard
+        let rootViewController: UIViewController?
```

Comment:
Tady by to mohlo byt jen storyboard name, ten init controlleru je pak pokazde stejny

Reply:
Neni. Ze storyboardu `ForceUpdate` beru 2 ruzne view controllery podle jmena (a pribude treti).